sass/_functions.scss

```diff
@@ -0,0 +1,11 @@
+@function calc-rem($size) {
+    @return ($size / $size-body) * 1rem;
+}
+
+@function darken-color($color) {
```

Comment:
Why saturate and darken vs just darken?

Reply:
Sincerely, I don't remember exactly. I've copied few of these helpers from other projects. The idea I think was that darken/lighten are more pleasant when are combined with saturate/desaturate, but probably is super questionable. 😇 

Here, same as with font-sizes, the ideal would be to just have a set of color tokens and choose from there, instead of using these helpers.

Maybe there is a designer in the community willing to prepare a design system… 😅

Reply:
Gotcha, I was more worried about the name since it's easy to confused `darken()` with `darken-color()`. Perhaps a more descriptive name?

Reply:
The name certainly is not very inspired. 😓  In other projects I'm using the Dart implementation which uses the new Sass modules, Zola uses the older `libsass` (which rises some questions about how that will be maintained, but that's another issue) and had to change the implementation… This is cleaner, but unsupported:

```
@use 'sass:color';

@function darken($color) {
  @return color.scale($color, $lightness: -5%, $saturation: 5%);
}
```

Anyway, to the point, I might remove these and use some color variables instead.

Reply:
I've removed the color functions. The solutions here should to use a palette… but let's post-pone that. ^_^